frontend/integration-tests/tests/crd-extensions.scenario.ts

```diff
@@ -207,7 +207,7 @@ describe('CRD extensions', () => {
       await yamlView.setEditorContent(safeDump(newContent));
       expect(yamlView.getEditorContent()).toContain(`location: ${location}`);
       await yamlView.saveButton.click();
-      await browser.wait(until.visibilityOf(crudView.successMessage), 1000);
+      await browser.wait(until.visibilityOf(crudView.successMessage), BROWSER_TIMEOUT);
```

Comment:
Is there a reason we need to change this? If it's too short, I would just use the default by leaving the second param off.

Reply:
It was too short, at least once, in my setup, and the change made the difference. Will test again without the 2nd param.

Reply:
Dropped the 2nd param, and verified FF worked.